### ISSUE TITLE
[QST] blazingsql on pypi ?

### ISSUE THREAD

**Issue by Imaclean74** [BlazingDB/pyBlazing#52]
I see the project has a setup.py and there is mention of conda on the last commit. However its not yet published to pypi. Is this planned ?

**Comment by felipeblazing** [comment]
It is but it is still a work in progress. Might be another week before it becomes a reality.

**Comment by felipeblazing** [comment]
And the plot thickens. We are debating whether its possible and will get back with an answer. Sorry its not so clear cut there are lots of interesting things issues that pop up when working with gpus and services for us.

**Comment by Imaclean74** [comment]
thanks @felipeblazing . I'm curious - is there some technical blocker ? or just a matter of someone putting the package together and publishing to pypi ?

**Comment by kkraus14** [comment]
@Imaclean74 I'm one of the maintainers of RAPIDS and work on the packaging side of things quite a bit. There are significant technical blockers, especially in respects to handling the CUDA toolkit and other native libraries that BlazingDB depend on. This limits their options to static linking everything (including the CUDA libraries, RAPIDS libraries, and Arrow libraries), which, in addition to massively blowing up the binary size, introduces issues as far as symbol conflicts if the user gets mismatched versions and loads one library before the other.

I wrote a brief blog explaining why we decided to drop support for pypi packages here: https://medium.com/rapids-ai/rapids-0-7-release-drops-pip-packages-47fc966e9472.
